macros/src/actix.rs

```diff
@@ -1093,6 +1115,42 @@ impl SerdeRename {
     }
 }
 
+/// Serde skip (https://serde.rs/variant-attrs.html)
+/// Never serialize or deserialize this variant.
+/// There are other variants available (skip_serializing,skip_deserializing) though it's not clear
+/// how this should be handled since we use the same Schema for Ser/DeSer
+struct SerdeSkip;
+
+impl SerdeSkip {
+    /// Traverses the field attributes and returns whether the field should be skipped or not
+    /// dependent on finding the `#[serde(skip]` attribute.
+    fn from_field_attrs(field_attrs: &[Attribute]) -> bool {
```

Comment:
`exists` is a better name, like what we have in the `SerdeFlatten` struct.

Reply:
it is indeed, I've fixed it